internal/environment/environment.go

```diff
@@ -65,9 +65,6 @@ func (p Paths) IndexBase() string {
 // e.g. {BasePath}/index/
 func (p Paths) IndexPath(name string) string {
 	if _, ok := os.LookupEnv(constants.EnableMultiIndexSwitch); ok {
-		if name == "" {
-			return filepath.Join(p.base, "index", constants.DefaultIndexName)
```

Comment:
Now we are not using `constants.DefaultIndexName` in all the places with a literal `"default"`. I think it would be better to do so, because it makes it even easier to find all usages.

Reply:
I totally forgot about that while I was adding this. I'll make that change.